### DISCUSSION TITLE
How I can add buttons and lists.

### DISCUSSION THREAD

**Discussion by talhag3** [WhiskeySockets/Baileys#433] [General Questions]
I'm working on Intractive buttons and lists/sections. Messages not visible after send no error while sending.

I'm following the documentation code.

**Comment by bruno918** [comment]
As far as I know, they no longer work, but maybe someone knows something...

**Comment by shizothetechie** [comment]
> I'm working on Intractive buttons and lists/sections. Messages not visible after send no error while sending.
> 
> I'm following the documentation code.  

Buttons, lists & Hydrated Buttons that are send using Baileys will no longer visible on WhatsApp as WhatsApp Blocked sending this through Baileys. 🙃
